app/Http/Middleware/TrustProxies.php

```diff
@@ -2,7 +2,7 @@
 
 namespace App\Http\Middleware;
 
-use Illuminate\Http\Middleware\TrustProxies as Middleware;
+use Fideloper\Proxy\TrustProxies as Middleware;
 use Illuminate\Http\Request;
```

Comment:
Please tell me why you make change?

Reply:
Hmm... I didn't realize I had changed it before. Maybe the reason comes from when I did some commands for updating Laravel or something like that. I had turned back to "Illuminate\Http\Middleware".